test/unit/module/core/first_true.cpp

```diff
@@ -103,7 +103,7 @@ TTS_CASE_TPL("Check eve::first_true", eve::test::simd::all_types)
     }
 
     x = true;
-    TTS_EQUAL(eve::first_true[eve::ignore_first(T::size())](x), std::nullopt);
+    TTS_EQUAL(eve::first_true[eve::ignore_last(T::size())](x), std::nullopt);
```

Comment:
why the change ? Was the test wrong since before, why didn't we found out ?

Reply:
It was created as a copy paste. This one I originally forgot to change